models/openacademy_course.py

```diff
@@ -10,8 +11,13 @@ class Course(models.Model):
     session_ids = fields.One2many("openacademy.session", string="Sessions", inverse_name="course_id")
 
     _sql_constraints = [
-        ("description_and_title_different", "CHECK(name != description)", "The title and description of the course must be different."),
-        ("unique_course_name", "UNIQUE(name)", "The name of the course must be unique.")
+        ("description_and_title_different",
```

Comment:
I agree with these changes, but please create a new PR to solve lints, as I commented before. That way, they could be merged immediately and we don't add unrelated changes on this PR/commit